tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/main.py

```diff
@@ -103,6 +110,7 @@ def main(event: Dict, context):    # pylint: disable=unused-argument
     prefix_to_load = removesuffix(object_id, SUCCESS_FILENAME)
     gsurl = f"gs://{bucket_id}/{prefix_to_load}"
     gcs_client = storage.Client(client_info=CLIENT_INFO)
+    project = gcs_client.project
     bkt = gcs_client.lookup_bucket(bucket_id)
     success_blob: storage.Blob = bkt.blob(object_id)
```

Comment:
Check and raise error if bkt is None before calling blob()

Reply:
goooood point mypy should have caught this if there was a proper typehint `lookup_bucket() -> Optional[storage.Bucket]`.

Reply:
Opened https://github.com/googleapis/python-storage/issues/318

Reply:
implemented this in a wrapper function and added local type hint.